src/sparseml/pytorch/torchvision/export_onnx.py

```diff
@@ -134,24 +146,49 @@ def main(
     interpolation: str,
     img_resize_size: int,
     img_crop_size: int,
+    deeplake_data_url: Optional[str],
+    deeplake_token: Optional[str],
+    deeplake_label_column: Optional[str],
+    deeplake_image_column: Optional[str],
 ):
     """
     SparseML torchvision integration for exporting image classification models to
     onnx along with sample inputs and outputs
     """
 
     save_dir.mkdir(parents=True, exist_ok=True)
+    mean = (0.485, 0.456, 0.406)
+    std = (0.229, 0.224, 0.225)
+    resize_size = 256
+    interpolation = InterpolationMode.BILINEAR
 
-    dataset = torchvision.datasets.ImageFolder(
-        dataset_path / "val",
-        presets.ClassificationPresetEval(
-            crop_size=img_crop_size,
-            resize_size=img_resize_size,
-            interpolation=InterpolationMode(interpolation),
-        ),
+    tform = deeplake_transforms.Compose(
+        [
+            deeplake_transforms.RandomRotation(20),  # Image augmentation
```

Comment:
why do we need to specify image augmentation for onnx export? Those two things are not related.

Reply:
also, I would move the whole deeplake logic into a helper function, as not to overclutter this script.